plugin-defaults/trace/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_library(DefaultTracing STATIC ${CMAKE_CURRENT_LIST_DIR}/trace/default-trace.c)
+target_include_directories(DefaultTracing PRIVATE ${CMAKE_CURRENT_LIST_DIR}/../../include/plugin-apis)
```

Comment:
OK, no problem. Originally I was confused when you said there was a cycle because I think of these APIs as being upstream of both the core and tracing, but when you characterize these as being part of `core` I see where you are coming from.

I can make them a dedicated header-only CMake target then.